client/my-sites/theme/main.jsx

```diff
@@ -644,9 +644,8 @@ export default connect(
 	( state, { id } ) => {
 		const selectedSite = getSelectedSite( state );
 		const siteSlug = selectedSite ? getSiteSlug( state, selectedSite.ID ) : '';
-		const isJetpack = selectedSite && isJetpackSite( state, selectedSite.ID );
-		const isWpcomTheme = selectedSite ? isThemeWpcom( state, id, selectedSite.ID ) : true;
-		const siteIdOrWpcom = ( isJetpack && ! isWpcomTheme ) ? selectedSite.ID : 'wpcom';
+		const isWpcomTheme = !! getTheme( state, 'wpcom', id );
+		const siteIdOrWpcom = ( selectedSite && ! isWpcomTheme ) ? selectedSite.ID : 'wpcom';
```

Comment:
> This is a bit redundant: We're already using getTheme with wpcom in the isWpcom definition on the line right before this one, and now, we're getting it again if it's falsey

Yes, this was troubling me a little. A question: is there any downside to multiple calls to `getTheme()`?